internal/app/grpcclient/impl.go

```diff
@@ -132,9 +132,12 @@ func (i *invoker) unary(
 }
 
 func (i *invoker) serverStream(
-	ctx context.Context,
+	c context.Context,
 	ccr *v1.ClientCompatRequest,
 ) (result *v1.ClientResponseResult, retErr error) {
+	ctx, ctxCancel := internal.WrapContext(c)
```

Comment:
Here and throughout.
```suggestion
	ctx, cancel := context.WithCancel(ctx)
```